MediaBrowser.Model/Configuration/LibraryOptions.cs

```diff
@@ -103,6 +110,17 @@ public LibraryOptions()
 
         public string[] LyricFetcherOrder { get; set; }
 
+        [DefaultValue(false)]
+        public bool PreferNonstandardArtistsTag { get; set; }
+
+        [DefaultValue(false)]
+        public bool UseCustomTagDelimiters { get; set; }
+
+        [DefaultValue(typeof(LibraryOptions), nameof(_defaultTagDelimiters))]
+        public char[] CustomTagDelimiters { get; set; }
```

Comment:
Late to the party but two issues here:

1. The default value causes an invalid OpenAPI spec - probably just remove it
2. The `char` type is invalid for OpenAPI, needs to be string

Reply:
We could add a custom filter for swashbuckle that maps the char type to string with min and max length set to 1. but that requires api generators to support that and interpret it as char.

Reply:
That is a problem because we only want a single character for this option. This means server will need to have a private field to return the sanitized results, and the default value is an array of char so I'm not quite sure why that does not work.

Reply:
The array part is not a problem - that works fine. The issue with using char is that there is no char type in OpenAPI.

The more important issue here is the default value though, that one is preventing our TypeScript SDK from building as `null` (the value emitted in the OpenAPI schema) is not a valid default for an array.